orga/app/templates/components/routes/join-request-form.hbs

```diff
@@ -0,0 +1,72 @@
+<div class="join-request-form">
+  {{#if this.hasErrorMessage}}
+    <div id="join-request-form-error-message" class="join-request-form__error-message error-message">{{this.errorMessage}}</div>
+  {{/if}}
+
+  <p class="join-request-form__information">Tous les champs sont obligatoires.</p>
+
+  <form>
+
+    <div class="input-container">
+      <label for="uai" class="label">UAI/RNE de l'établissement</label>
+      {{#if this.validation.uai.hasError}}
+        <div class="alert-input alert-input--error" role="alert">{{this.validation.uai.message}}</div>
+      {{/if}}
+      <Input
+        @id="uai"
+        @name="uai"
+        @type="text"
+        @class="input {{if this.validation.uai.hasError 'input--error'}}"
+        @value={{this.uai}}
+        @required='true'
+        {{on "focusout" (fn this.validateInput "uai" this.uai)}}
```

Comment:
<Input
        id="uai"
        name="uai"
        @type="text"
        class="input {{if this.validation.uai.hasError 'input--error'}}"
        @value={{this.uai}}
        required={{true}}

Reply:
Done